### PR TITLE
feat(radar): add onClick handler

### PR DESCRIPTION
**Pull request by Nino3103**
## Radar Chart : add onClick handler

- added onClick props in Radar props
- adjusted the types to accept onClick in Radar props
- added a story in Radar storybook with onClick

## Video
https://github.com/plouc/nivo/assets/48714971/edc20b8a-df04-4cdb-85d2-49efb8b8ce90

### CODE REVIEW

**Comment by charles-glanceable** [comment]
We are really looking forward to this new feature! Thank you 👍

**Review by plouc — CHANGES_REQUESTED** [review]
Overall, LGTM, thank you.

However the property should be added to the website (I'm not sure we need a story), also it would be good to add some tests for this new feature.

**Comment by Nino3103** [comment]
Thank you for your answer, I will add it to the website and add some tests 👍 Should I remove the story ?

**Comment by Nino3103** [comment]
Hello @plouc , I added a radar test and updated the website


https://github.com/plouc/nivo/assets/48714971/d5653a24-3606-4681-9707-d077aa178fa1

File: packages/radar/tests/Radar.test.tsx [hunk 1]
```diff
@@ -222,3 +224,22 @@ describe('legend', () => {
         expect(wrapper.find(BoxLegendSvg).find('text').at(3).text()).toBe(customLabels[1].B)
     })
 })
+
+describe('interactivity', () => {
+    it('should support onClick handler', async () => {
+        const onClick = jest.fn()
+        // const wrapper = mount(<Radar<TestDatum> {...baseProps} />)
```
Comment (plouc) [inline packages/radar/tests/Radar.test.tsx:231]: Should be removed :)

File: storybook/stories/radar/Radar.stories.tsx [hunk 2]
```diff
@@ -227,3 +227,14 @@ export const WithPatterns: Story = {
 export const WithRotation: Story = {
     render: args => <Radar {...commonProperties} curve={args.curve} rotation={36} />,
 }
+
```
Comment (plouc) [inline storybook/stories/radar/Radar.stories.tsx:230]: I think it can be removed as it's now included to the website, less to maintain :)

**Review by plouc — CHANGES_REQUESTED** [review]

File: packages/radar/tests/Radar.test.tsx (hunk 1 above)
Comment (Nino3103) [reply packages/radar/tests/Radar.test.tsx]: I removed the commented line (231) :)

**Comment by Nino3103** [comment]
@plouc I've resync with master, tests are passing now 👍

**Comment by plouc** [comment]
@Nino3103, thank you for your contribution!

**Pull request merged**